sparse/coo/common.py

```diff
@@ -637,6 +637,7 @@ def roll(a, shift, axis=None):
         Output array, with the same shape as a.
     """
     from .core import COO, as_coo
+    import collections
```

Comment:
Can you do this at the top of the file?

Reply:
yep. done in latest commit.

Reply:
And remove it from here?

Reply:
https://github.com/ahwillia/sparse/commit/a6339db9498202fcee4d7561c96783bee26f6a3a

Reply:
Ah, my mistake. Did you push it to the repo, are you sure it isn't local?

Reply:
![image](https://user-images.githubusercontent.com/636625/41869452-e52d7f7a-786d-11e8-9b41-9a2e2d0007e4.png)

Reply:
I get the same. I'm assuming GitHub is migrating stuff, and since it uses microservices, one thing is breaking at a time. Guess we'll just have to wait until things catch up and the tests run.

Reply:
Could you possibly make a minor change and re-push to the repo? Possibly squash all the commits?